.github/workflows/deploy-master.yml

```diff
@@ -0,0 +1,49 @@
+name: deploy-master
+
+on:
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+env:
+  SECRET1: ${{ secrets.fh }}
+  SECRET2: ${{ secrets.fu }}
+  SECRET3: ${{ secrets.fp }}
+  DDOMAIN: absonline.educationhost.cloud
+
+jobs:
+  deploy-master-job:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install lftp
+      run: sudo apt install -y --no-install-recommends lftp >/dev/null
+
+    - name: Write the FTP reverse mirror command to a file
+      run: |
+        rm -rfv /dev/shm/somecommand
+        echo '#!/bin/bash'							 >/dev/shm/somecommand
+        echo "lftp $SECRET1 -e \""					>>/dev/shm/somecommand
+        echo ' set ftp:ssl-force true'				>>/dev/shm/somecommand
+        echo ' set ssl:verify-certificate true'		>>/dev/shm/somecommand
+        echo ' set ftp:list-options -a'				>>/dev/shm/somecommand
+        echo " set ftp:anon-user $SECRET2"			>>/dev/shm/somecommand
+        echo " set ftp:anon-pass $SECRET3"			>>/dev/shm/somecommand
+        echo ' mirror -cesRLv \'					>>/dev/shm/somecommand
+        echo '   -x^\.git/$ -x^\.github/$ \'		>>/dev/shm/somecommand
+        echo '   -X **/*.{MD,mD,Md,md} \'			>>/dev/shm/somecommand
+        echo "   . domains/$DDOMAIN/public_html"	>>/dev/shm/somecommand
+        echo ' quit'								>>/dev/shm/somecommand
+        echo '"'									>>/dev/shm/somecommand
+        chmod 0500 /dev/shm/somecommand
+
+    - name: Fetch the last commit from repository
+      uses: actions/checkout@v2
+      with:
+        persist-credentials: false
+        clean: false
+
+    - name: Change to repo directory and run command from the file
+      run: |
+        cd "$GITHUB_WORKSPACE"
+        sh /dev/shm/somecommand
```

Comment:
Lol, silly me again 🙃
I set the interpreter to `bash` above (although not required) and ran the file with `sh` 😆